atlp-rwanda/atlp-pulse-mobile
# #44 Improve registration flow

### What does this PR do?

This PR includes redirect implementations similar to the frontend that handles redirection in case a user selects the app to handle redirection.

### Description of Task to be completed?

Once a trainee is invited in an organization, they receive an email with a link to register. Once they click on the link they will be prompted to choose either the website or app to handle opening URLs, If they choose the app they should redirect to the app and the app will handle the redirection.

### How should this be manually tested?

- Open the app and click on `Get started`
- That links to the redirection to the login age (check sources)
- The app will redirect to the login page
- If you click back it shouldn't go to redirect page which can cause infinite redirections. 

### Screenshots
N/A
